tests/Test/AbstractFixerWithAliasedOptionsTestCase.php

```diff
@@ -0,0 +1,87 @@
+<?php
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PhpCsFixer\Tests\Test;
+
+use PhpCsFixer\Fixer\ConfigurationDefinitionFixerInterface;
+use PhpCsFixer\FixerConfiguration\AliasedFixerOption;
+
+/**
+ * @author ntzm
+ *
+ * @internal
+ */
+abstract class AbstractFixerWithAliasedOptionsTestCase extends AbstractFixerTestCase
+{
+    /**
+     * @var null|\PhpCsFixer\Fixer\ConfigurableFixerInterface
+     */
+    private $fixerWithAliasedConfig;
+
+    protected function tearDown()
+    {
+        parent::tearDown();
+
+        unset($this->fixerWithAliasedConfig);
```

Comment:
Set the property to `null` rather than using `unset()`, which drops the property itself.